### PR TITLE
Add requirements section to run and develop in the project

### PR DESCRIPTION
**Pull request by mohammad4kh**
Add required packages to run and develop the frontend project and how to install the node packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,41 @@ An Open Source Social Media Platform and Public Forum for Questions and Discussi
...
+      $ sudo apt install nodejs
+      $ sudo apt install npm
+
+- #### Other Operating Systems
+  You can find more information about the installation on the [official Node.js website](https://nodejs.org/) and the [official NPM website](https://npmjs.org/).
+
+  If the installation was successful, you should be able to run the following command.
+    
+      $ node --version
+      v12.18.2
```
Comment (codyseibert) [inline README.md:33]: We at least want node version v14.x, could you update this to reflect the latest LTS stable version of node?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,41 @@ An Open Source Social Media Platform and Public Forum for Questions and Discussi
...
+      
+      $ npm --version
+      6.14.8
+
+- #### For Yarn installation
+  After installing node, this project will need yarn too, so just run the following command.
```
Comment (codyseibert) [inline README.md:39]: we actually don't require yarn.  I think we are using NPM by default.  Yarn is optional if you rather develop with yarn.

File: README.md (hunk 1 above)
Comment (mohammad4kh) [reply README.md:33]: Ok I'll update the version example to show the latest LTS and emphasis that the least version is v14.x

File: README.md (hunk 2 above)
Comment (mohammad4kh) [reply README.md]: PR is updated now.

**Review by codyseibert — APPROVED** [review]
Looks good!

**Pull request merged**